core/src/com/group66/game/cannon/BallManager.java

```diff
@@ -32,11 +32,14 @@
 
 	/** The static ball list. */
 	private ArrayList<Ball> ballStaticList = new ArrayList<Ball>();
-
-	/** The static ball dead list. */
+	
+	/**  The ball to be added to static List. */
```

Comment:
The previous comment "/** The static ball deadlist" is more informative and more clear.